lib/Mojolicious/Sessions.pm

```diff
@@ -150,6 +151,18 @@ A callback used to deserialize sessions, defaults to L<Mojo::JSON/"j">.
 Use encrypted session cookies instead of merely cryptographically signed ones. Note that this attribute is
 B<EXPERIMENTAL> and might change without warning!
 
+=head2 partitioned
+
+  my $bool  = $sessions->partitioned;
+  $sessions = $sessions->partitioned($bool);
+
+Set the partitioned flag on all session cookies, this is used in accordance to the L<CHIPS ammendment to
+RFC 6265|https://www.ietf.org/archive/id/draft-cutler-httpbis-partitioned-cookies-00.html>.
+Note that this attribute is B<EXPERIMENTAL> because even though most commonly used browsers support the
+feature, the specification is still a draft.
```

Comment:
Please use the same format as other features do for marking them as experimental.